examples/mnist/Instructions.md

```diff
@@ -0,0 +1,25 @@
+<p align="center"><b>How to test example MNIST train/test programs</b><p align="center">
```

Comment:
Changing the filename from `Instructions.md` to `how_to_build.md` might be good. What do you think?